<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BuzzMainWindow</class>
 <widget class="QMainWindow" name="BuzzMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1231</width>
    <height>1022</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="RPMlbl">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>520</y>
      <width>62</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>RPM: </string>
    </property>
   </widget>
   <widget class="QPushButton" name="ChuteDrop">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>350</y>
      <width>114</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Chute Drop! </string>
    </property>
   </widget>
   <widget class="QPushButton" name="AirHold">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>114</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Airport Hold</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ComHold">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>114</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Comms Hold</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="BigText">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>770</y>
      <width>471</width>
      <height>161</height>
     </rect>
    </property>
    <property name="plainText">
     <string>mav_output / console here</string>
    </property>
   </widget>
   <widget class="QLabel" name="RPMlbl_2">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>520</y>
      <width>62</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>RPM</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Miss1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>114</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Do Mission 1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Miss2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>240</y>
      <width>114</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Do Mission 2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Miss3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>270</y>
      <width>114</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Do Mission 3</string>
    </property>
   </widget>
   <widget class="QLabel" name="chutedrop_out">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>360</y>
      <width>62</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>operating.</string>
    </property>
   </widget>
   <widget class="QLabel" name="bottledrop_out">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>330</y>
      <width>62</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>operating</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BottleDrop">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>320</y>
      <width>114</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Bottle Drop! </string>
    </property>
   </widget>
   <widget class="QLabel" name="airhold_out">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>130</y>
      <width>62</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>operating</string>
    </property>
   </widget>
   <widget class="QLabel" name="comhold_out">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>160</y>
      <width>62</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>operating</string>
    </property>
   </widget>
   <widget class="QLabel" name="miss1_out">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>220</y>
      <width>62</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>operating</string>
    </property>
   </widget>
   <widget class="QLabel" name="miss2_out">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>250</y>
      <width>62</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>operating</string>
    </property>
   </widget>
   <widget class="QLabel" name="miss3_out">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>280</y>
      <width>62</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>operating</string>
    </property>
   </widget>
   <widget class="QSlider" name="LhorizontalSlider">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>840</y>
      <width>160</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1000</number>
    </property>
    <property name="maximum">
     <number>2000</number>
    </property>
    <property name="value">
     <number>1500</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="LverticalSlider">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>770</y>
      <width>22</width>
      <height>160</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1000</number>
    </property>
    <property name="maximum">
     <number>2000</number>
    </property>
    <property name="value">
     <number>1500</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QSlider" name="RhorizontalSlider">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>840</y>
      <width>160</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1000</number>
    </property>
    <property name="maximum">
     <number>2000</number>
    </property>
    <property name="value">
     <number>1500</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="RverticalSlider">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>770</y>
      <width>22</width>
      <height>160</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1000</number>
    </property>
    <property name="maximum">
     <number>2000</number>
    </property>
    <property name="value">
     <number>1500</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="Connect">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>50</y>
      <width>114</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Re-Connect!</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Manual">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>400</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Manual</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Auto">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>400</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Auto</string>
    </property>
   </widget>
   <widget class="QPushButton" name="RTL">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>420</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>RTL</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Stabilise">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>420</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Stabilise</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Circle">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>440</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Circle</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Guided">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>440</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Guided</string>
    </property>
   </widget>
   <widget class="QWebView" name="webView">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>-20</y>
      <width>951</width>
      <height>761</height>
     </rect>
    </property>
    <property name="url">
     <url>
      <string>file:///Users/buzz/Documents/Arduino/MAVProxy/mymap.html</string>
     </url>
    </property>
   </widget>
   <widget class="QPushButton" name="Exit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>161</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Exit App Now!</string>
    </property>
   </widget>
   <widget class="QLabel" name="modelbl">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>550</y>
      <width>62</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Mode:</string>
    </property>
   </widget>
   <widget class="QLabel" name="mode">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>550</y>
      <width>62</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Mode</string>
    </property>
   </widget>
   <widget class="QLabel" name="GPSlbl">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>570</y>
      <width>62</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>GPS:</string>
    </property>
   </widget>
   <widget class="QLabel" name="GPS">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>570</y>
      <width>62</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>GPS</string>
    </property>
   </widget>
   <widget class="QLabel" name="Vcclbl">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>590</y>
      <width>62</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Vcc:</string>
    </property>
   </widget>
   <widget class="QLabel" name="Vcc">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>590</y>
      <width>62</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Vcc</string>
    </property>
   </widget>
   <widget class="QLabel" name="Radiolbl">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>610</y>
      <width>62</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Radio:</string>
    </property>
   </widget>
   <widget class="QLabel" name="Radio">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>610</y>
      <width>62</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Radio</string>
    </property>
   </widget>
   <widget class="QLabel" name="Headinglbl">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>630</y>
      <width>62</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Heading:</string>
    </property>
   </widget>
   <widget class="QLabel" name="Heading">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>630</y>
      <width>62</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Heading</string>
    </property>
   </widget>
   <widget class="QLabel" name="Altlbl">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>650</y>
      <width>62</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Alt:</string>
    </property>
   </widget>
   <widget class="QLabel" name="Alt">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>650</y>
      <width>62</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Alt</string>
    </property>
   </widget>
   <widget class="QLabel" name="AirSpeedlbl">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>670</y>
      <width>62</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>AirSpeed:</string>
    </property>
   </widget>
   <widget class="QLabel" name="AirSpeed">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>670</y>
      <width>62</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>AirSpeed</string>
    </property>
   </widget>
   <widget class="QLabel" name="GPSSpeedlbl">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>690</y>
      <width>62</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>GPSSpeed:</string>
    </property>
   </widget>
   <widget class="QLabel" name="GPSSpeed">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>690</y>
      <width>62</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>GPSSpeed</string>
    </property>
   </widget>
   <widget class="QLabel" name="Rolllbl">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>710</y>
      <width>62</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Roll:</string>
    </property>
   </widget>
   <widget class="QLabel" name="Roll">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>710</y>
      <width>62</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Roll</string>
    </property>
   </widget>
   <widget class="QLabel" name="Pitchlbl">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>730</y>
      <width>62</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Pitch:</string>
    </property>
   </widget>
   <widget class="QLabel" name="Pitch">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>730</y>
      <width>62</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Pitch</string>
    </property>
   </widget>
   <widget class="QLabel" name="WPlbl">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>750</y>
      <width>62</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>WP:</string>
    </property>
   </widget>
   <widget class="QLabel" name="WP">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>750</y>
      <width>62</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>WP</string>
    </property>
   </widget>
   <widget class="QLabel" name="lalbl">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>470</y>
      <width>62</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>lattitude:</string>
    </property>
   </widget>
   <widget class="QLabel" name="la">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>470</y>
      <width>62</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>lattitude</string>
    </property>
   </widget>
   <widget class="QLabel" name="lllbl">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>490</y>
      <width>62</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>longitude:</string>
    </property>
   </widget>
   <widget class="QLabel" name="ll">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>490</y>
      <width>62</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>longitude</string>
    </property>
   </widget>
   <widget class="BuzzGraphicsView" name="LeftGraphicsView">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>770</y>
      <width>180</width>
      <height>180</height>
     </rect>
    </property>
   </widget>
   <widget class="BuzzGraphicsView" name="RightGraphicsView">
    <property name="geometry">
     <rect>
      <x>1040</x>
      <y>770</y>
      <width>180</width>
      <height>180</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="Surfaceslbl">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>750</y>
      <width>62</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Surfaces</string>
    </property>
   </widget>
   <widget class="QLabel" name="Steeringlbl">
    <property name="geometry">
     <rect>
      <x>1010</x>
      <y>750</y>
      <width>62</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Steering</string>
    </property>
   </widget>
   <widget class="QLabel" name="infobar">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>950</y>
      <width>1181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Info Bar</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1231</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QWebView</class>
   <extends>QWidget</extends>
   <header>QtWebKit/QWebView</header>
  </customwidget>
  <customwidget>
   <class>BuzzGraphicsView</class>
   <extends>QGraphicsView</extends>
   <header>BuzzGraphicsView</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>ComHold</sender>
   <signal>clicked()</signal>
   <receiver>BuzzMainWindow</receiver>
   <slot>comshold()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>76</x>
     <y>186</y>
    </hint>
    <hint type="destinationlabel">
     <x>143</x>
     <y>101</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Miss1</sender>
   <signal>clicked()</signal>
   <receiver>BuzzMainWindow</receiver>
   <slot>miss1()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>101</x>
     <y>247</y>
    </hint>
    <hint type="destinationlabel">
     <x>141</x>
     <y>153</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Miss2</sender>
   <signal>clicked()</signal>
   <receiver>BuzzMainWindow</receiver>
   <slot>miss2()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>105</x>
     <y>272</y>
    </hint>
    <hint type="destinationlabel">
     <x>134</x>
     <y>186</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Miss3</sender>
   <signal>clicked()</signal>
   <receiver>BuzzMainWindow</receiver>
   <slot>miss3()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>110</x>
     <y>306</y>
    </hint>
    <hint type="destinationlabel">
     <x>139</x>
     <y>222</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>BottleDrop</sender>
   <signal>clicked()</signal>
   <receiver>BuzzMainWindow</receiver>
   <slot>bottleaway()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>105</x>
     <y>356</y>
    </hint>
    <hint type="destinationlabel">
     <x>136</x>
     <y>266</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ChuteDrop</sender>
   <signal>clicked()</signal>
   <receiver>BuzzMainWindow</receiver>
   <slot>chuteaway()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>101</x>
     <y>384</y>
    </hint>
    <hint type="destinationlabel">
     <x>134</x>
     <y>306</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>LhorizontalSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>BuzzMainWindow</receiver>
   <slot>rud(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>550</x>
     <y>883</y>
    </hint>
    <hint type="destinationlabel">
     <x>539</x>
     <y>575</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Auto</sender>
   <signal>clicked()</signal>
   <receiver>BuzzMainWindow</receiver>
   <slot>auto()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>76</x>
     <y>434</y>
    </hint>
    <hint type="destinationlabel">
     <x>95</x>
     <y>357</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Manual</sender>
   <signal>clicked()</signal>
   <receiver>BuzzMainWindow</receiver>
   <slot>manual()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>124</x>
     <y>436</y>
    </hint>
    <hint type="destinationlabel">
     <x>193</x>
     <y>373</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>RTL</sender>
   <signal>clicked()</signal>
   <receiver>BuzzMainWindow</receiver>
   <slot>rtl()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>28</x>
     <y>457</y>
    </hint>
    <hint type="destinationlabel">
     <x>9</x>
     <y>406</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Stabilise</sender>
   <signal>clicked()</signal>
   <receiver>BuzzMainWindow</receiver>
   <slot>stabilise()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>137</x>
     <y>453</y>
    </hint>
    <hint type="destinationlabel">
     <x>193</x>
     <y>409</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Circle</sender>
   <signal>clicked()</signal>
   <receiver>BuzzMainWindow</receiver>
   <slot>circle()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>45</x>
     <y>477</y>
    </hint>
    <hint type="destinationlabel">
     <x>39</x>
     <y>447</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Guided</sender>
   <signal>clicked()</signal>
   <receiver>BuzzMainWindow</receiver>
   <slot>guided()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>120</x>
     <y>477</y>
    </hint>
    <hint type="destinationlabel">
     <x>163</x>
     <y>445</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>LverticalSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>BuzzMainWindow</receiver>
   <slot>thr(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>626</x>
     <y>818</y>
    </hint>
    <hint type="destinationlabel">
     <x>575</x>
     <y>534</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>RhorizontalSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>BuzzMainWindow</receiver>
   <slot>ail(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>739</x>
     <y>871</y>
    </hint>
    <hint type="destinationlabel">
     <x>701</x>
     <y>585</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>RverticalSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>BuzzMainWindow</receiver>
   <slot>ele(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>794</x>
     <y>809</y>
    </hint>
    <hint type="destinationlabel">
     <x>806</x>
     <y>558</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Exit</sender>
   <signal>clicked()</signal>
   <receiver>BuzzMainWindow</receiver>
   <slot>exitnow()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>134</x>
     <y>45</y>
    </hint>
    <hint type="destinationlabel">
     <x>191</x>
     <y>37</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Connect</sender>
   <signal>clicked()</signal>
   <receiver>BuzzMainWindow</receiver>
   <slot>reconnect()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>135</x>
     <y>86</y>
    </hint>
    <hint type="destinationlabel">
     <x>171</x>
     <y>81</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>airhold()</slot>
  <slot>comshold()</slot>
  <slot>miss1()</slot>
  <slot>miss2()</slot>
  <slot>miss3()</slot>
  <slot>bottleaway()</slot>
  <slot>chuteaway()</slot>
  <slot>ail(int)</slot>
  <slot>ele(int)</slot>
  <slot>thr(int)</slot>
  <slot>rud(int)</slot>
  <slot>auto()</slot>
  <slot>rtl()</slot>
  <slot>manual()</slot>
  <slot>stabilise()</slot>
  <slot>guided()</slot>
  <slot>circle()</slot>
  <slot>exitnow()</slot>
  <slot>reconnect()</slot>
  <slot>pwm_direction(bool)</slot>
 </slots>
</ui>
